app/controllers/api/v1/stats/channels_controller.rb

```diff
@@ -0,0 +1,48 @@
+class Api::V1::Stats::ChannelsController < Api::BaseController
+
+def channels
+  channels = Channel.all.map { |channel| channel.id }
+  data = JSON.pretty_generate(channels)
+
+  render(json: data)
+
+end
+
+def channels_uuid
+
+  channel = Channel.find_by_id(params[:uuid])
+
+  case channel.details_type
+  when "SiteChannelDetails"
+    channel_details = SiteChannelDetails.find_by_id(channel.details_id)
+    platform = "Site"
+    channel_id = channel_details.brave_publisher_id
+  when "YoutubeChannelDetails"
+    channel_details = YoutubeChannelDetails.find_by_id(channel.details_id)
+    platform = "Youtube"
+    channel_id = channel_details.youtube_channel_id
+  when "TwitterChannelDetails"
+    channel_details = TwitterChannelDetails.find_by_id(channel.details_id)
+    platform = "Twitter"
+    channel_id = channel_details.twitter_channel_id
+  when "TwitchChannelDetails"
+    channel_details = TwitchChannelDetails.find_by_id(channel.details_id)
+    platform = "Twitch"
+    channel_id = channel_details.twitch_channel_id
```

Comment:
I think we want `channel_details.channel_identifier` here and above:

```
irb(main):117:0> channel_details.channel_identifier
=> "youtube#channel:UCtsfHRe2WQnkNH5WYJWL-Yw"
irb(main):118:0> channel_details.youtube_channel_id
=> "UCtsfHRe2WQnkNH5WYJWL-Yw"
```

Reply:
Fixed, I also added the `url` property you mentioned in our call

question: I don't see `channel_identifier`, or `url` in `schema.rb` how did you know about those properties?

Reply:
@dlipeles they are methods on the models, not fields in the database

https://github.com/brave-intl/publishers/blob/8b6093cb8fc8487534e1e898d0a224ef00a0166c/app/models/youtube_channel_details.rb#L12-L14

Reply:
Ah very cool, thanks Nick